### DISCUSSION TITLE
Can I customize their power mode whenever the power cord is plugged in and the power cord is unplugged?

### DISCUSSION THREAD

**Discussion by Mi3sbbbi** [seerge/g-helper#1351] [General]
Can I customize their power mode whenever the power cord is plugged in and the power cord is unplugged?

**Comment by seerge** [comment]
@Mi3sbbbi app already saves mode separately for plugged / on battery states

**Reply by Mi3sbbbi** [reply]
Is there any way I can customize these insert/open state modes?

**Reply by seerge** [reply]
I advice you to check readme https://github.com/seerge/g-helper#readme

**Reply by Mi3sbbbi** [reply]
sorry I didn't get it right, I'm not referring to the WINDOWS power plan. It's Ghelper mode. For example, I set silent mode to low power, which makes my ROG more power efficient. I need it to automatically switch to silent mode when I unplug, and balance mode when I plug in.

**Reply by seerge** [reply]
As I have just answered app already saves last used mode per battery/plugged cases and turns them on every time you plug or unplug

**Reply by Mi3sbbbi** [reply]
Yes, but when I need to use enhanced mode in a certain situation, the application will remember my temporary need, which is not what I want, and I hope that when I set its mode, it will not remember my subsequent adjustments to the mode, even if it is multiple optional boxes, (Remember current mode) or (no longer remember, save subsequent battery/power last mode)
